.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,36 @@
+name: Feature Request
+description: Suggest an idea for improving RDF4J
+labels: [ "📶 enhancement", "triage" ]
```

Comment:
If we can get the github action to put new issues on the project board working again, I'll remove the "triage" tag from these templates again, as it would be redundant with how we use the "Inbox" column.